Yoast/wordpress-seo
# Move crawl cleanup to free

## Context


* We want to move the crawl optimization functionalities from Yoast SEO Premium to Yoast SEO.

## Summary


This PR can be summarized in the following changelog entry:

* Makes the crawl optimization functionalities accessible to users of Yoast SEO

## Relevant technical choices:

* `src/integrations/admin/crawl-settings-integration.php` has been copied from Premium and adapted.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Deactivate Yoast SEO Premium if it is active
* Go to `Yoast SEO` -> `Settings`
* In the sidebar, under the `Advanced` section, verify that no `Premium` badge is presents beside `Crawl optimization` 
* Click on `Crawl optimization` and verify
  * verify that all the toggles are active and not greyed-out
  * verify the functionalities work as intended (look at [this doc](https://docs.google.com/document/d/1wbrxt7H6-zX2aD5Pw8RTqqWU8IYYMw7La3HWgmU20N0/edit) for reference)
* Activate Yoast SEO Premium and verify everything still works

**Test multisite functionalities**
* Go to `My Sites` -> `Network Admin` -> `Dashboard`
* Go to `Yoast SEO` -> `General`
  * verify the tabs are as follows:  
  <img width="676" alt="Screenshot 2023-03-06 at 15 38 43" src="https://user-images.githubusercontent.com/68744851/223141825-e755b2c9-3982-4220-a6f5-cf78fe05c144.png">
* Go to the `Crawl settings` tab and set `Shortlinks` to `Disabled`
* Go to the admin area of one of the subsites
* Go to `Yoast SEO` -> `Settings` -> `Advanced` -> `Crawl optimization`
  * Verify  the first toggle `Remove Shortlinks` is disabled and appears as follows
  <img width="717" alt="Screenshot 2023-03-06 at 15 48 41" src="https://user-images.githubusercontent.com/68744851/223144442-3757b24b-f2de-4db1-8124-9cc91d921989.png">

#### Relevant test scenarios


### Test instructions for QA when the code is in the RC



## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

* 

## UI changes


## Other environments


## Documentation


## Quality assurance


## Innovation


Fixes https://github.com/Yoast/roadmap/issues/198
